runtime/gc_glue_java/HeapRegionDescriptorStandardExtension.hpp

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2023
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
```

Comment:
@amicic @RSalman 
This is an invalid SPDX line. See https://github.com/eclipse-openj9/openj9/issues/17429; please create a pull request to correct it.

Reply:
> @amicic @RSalman
> This is an invalid SPDX line. See #17429; please create a pull request to correct it.

@keithc-ca Thank you very much for spotting this. Fix is in
https://github.com/eclipse-openj9/openj9/pull/17489

Reply:
Thanks @dmitripivkine !